src/modules/Driver.ts

```diff
@@ -141,32 +184,54 @@ export class Driver {
     return response;
   }
 
-  /** Simulates the press and release of the specified key. */
-  async sendKeyPress({ keyPress, retries, params }: { keyPress: string; retries: number; params: Params }) {
-    const url = `${this.baseURL}/keypress/${keyPress}`;
+  async sendKey({ keyType, key, retries, params }: { keyType: string; key: string; retries: number; params: Params }) {
+    const url = `${this.baseURL}/key${keyType}/${key}`;
     const response = await http.basePOST({ url, retries, params });
     // Responses do not contain a response body, so response is just the status code. Anything in the 200's is considered successful.
     return response;
   }
 
+  /** Simulates the press and release of the specified key. */
+  async sendKeyPress({ keyPress, retries, params }: { keyPress: string; retries: number; params: Params }) {
+    return this.sendKey({ keyType: 'press', key: keyPress, retries, params });
+  }
+
+  /** Simulates the press down of the specified key. */
+  async sendKeyDown({ keyDown, retries, params }: { keyDown: string; retries: number; params: Params }) {
```

Comment:
do these need to be three separate functions?

Reply:
Ah yeah, I guess each of the `RokulRunnings` functions can use the base key send function from `Driver`

Reply:
Resolved in 25ff9b6